### PR TITLE
IPR demo fixes

### PR DESCRIPTION
**Pull request by qrayven**
- removed `offchainstorage` and moved to `parser`
- some debug messges
- `Makefile`
- error catching
- iota ebsi provider in `truffle` 
- add support for wallet service

### CODE REVIEW

File: lib/storage/offChainStorage.js [hunk 1]
```diff
@@ -1,38 +0,0 @@
...
-    const parsedCID = this.CID.parse(cid);
-    return await this.jsonHelia.get(parsedCID);
-  }
-
-  async stop() {
-    await this.helia.stop();
-  }
-}
-
-module.exports = { OffChainStorage };
```
Comment (miker83z) [inline lib/storage/offChainStorage.js:38]: @qrayven Why did you remove the IPFS storage? Don't you need it?

File: wallet/lib/walletService.ts [hunk 2]
```diff
@@ -0,0 +1,51 @@
+// so we need the http client to make a request to the wallet service
+// additionally the wallet service probably is protected by the JWT token so this needs to be passed as well
+
+
+export const DefaultWalletServiceBaseURL = "http://localhost:8081"
```
Comment (jmcanterafonseca-iota) [inline wallet/lib/walletService.ts:5]: where is this configured externally?

File: lib/storage/offChainStorage.js (hunk 1 above)
Comment (qrayven) [reply lib/storage/offChainStorage.js:38]: this has been moved to the Parser, as it caused import cycles.

File: wallet/lib/walletService.ts (hunk 2 above)
Comment (qrayven) [reply wallet/lib/walletService.ts:5]: this is configured in constructor here https://github.com/iotaledger/MPEG-21-Part-23-DLT-Manager/pull/2/files#diff-d671f4fa983aef3f2b3535e4ee97b908251541597486c6756e2e40e51226de56R27  So the assumption is the new web app will be used it as library, and this is up to web application configure the proper endpoint

**Review by miker83z — APPROVED** [review]
